curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -17,439 +17,464 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
 
 #### --text--
 
-Placeholder question
+Which html element allows a two-dimensional table with column and row?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<row>`
 
 ---
 
-Placeholder distractor 2
+`<matrix>`
 
 ---
 
-Placeholder distractor 3
+`<table r="3" c="4">`
 
 #### --answer--
 
-Placeholder answer
+`<table></table>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to specify row and header respectively?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<r><thead>`
 
 ---
 
-Placeholder distractor 2
+`<row><th>`
 
 ---
 
-Placeholder distractor 3
+`<tr><head>`
 
 #### --answer--
 
-Placeholder answer
+`<tr><th>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is `<td>` element?
 
 #### --distractors--
 
-Placeholder distractor 1
+Used to define a table data type
 
 ---
 
-Placeholder distractor 2
+Used to merge two columns
 
 ---
 
-Placeholder distractor 3
+Used to merge two rows
 
 #### --answer--
 
-Placeholder answer
+Used to define a table cell
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is colspan?
 
 #### --distractors--
 
-Placeholder distractor 1
+Define the length of a column in a table
 
 ---
 
-Placeholder distractor 2
+To remove the breakline between text in a column
 
 ---
 
-Placeholder distractor 3
+Used to define the number of column
 
 #### --answer--
 
-Placeholder answer
+Merges a cell across multiple columns
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+`<table>
+    <tr>
+        <td colspan="3">This cell spans across three rows</td>
+    </tr>
+</table>`
+How to make the span across two rows only?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<table>
+    <tr>
+        <td row="3">This cell spans across three rows</td>
+    </tr>
+</table>`
 
 ---
 
-Placeholder distractor 2
+`<table>
+    <tr rowspan="3">
+        <td colspan="3">This cell spans across three rows</td>
+    </tr>
+</table>`
 
 ---
 
-Placeholder distractor 3
+Not possible
 
 #### --answer--
 
-Placeholder answer
+`<table>
+    <tr>
+        <td rowspan="3">This cell spans across three rows</td>
+    </tr>
+</table>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Waht is the purpose of `<thead>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+Can be used interchangeably with `<th>`
 
 ---
 
-Placeholder distractor 2
+Defines a single header cell
 
 ---
 
-Placeholder distractor 3
+Essential for marking table headers
 
 #### --answer--
 
-Placeholder answer
+Groups table header rows 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+`<tfoot>` can be placed before or after `<tbody>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+False
 
 ---
 
-Placeholder distractor 2
+Only Before
 
 ---
 
-Placeholder distractor 3
+Only After
 
 #### --answer--
 
-Placeholder answer
+True
```

Comment:
An either-or question can't have true/false as answers

Reply:
```suggestion
Both before and after.
```
Maybe this?

Reply:
I think when it comes all together, they make sense.